src/index.tsx

```diff
@@ -4,23 +4,33 @@ import { Provider } from 'react-redux';
 
 import 'font-awesome-sass-loader';
 
+import {Config} from 'vega-lite/build/src/config';
 import { Data } from 'vega-lite/build/src/data';
 import {App} from './components/app';
 import { VOYAGER_CONFIG } from './constants';
 import { VoyagerConfig } from './models/config';
 import { configureStore } from './store';
 
 const store = configureStore();
-const config: VoyagerConfig = VOYAGER_CONFIG;
+let config: VoyagerConfig = VOYAGER_CONFIG;
```

Comment:
I think the demo is a bit unnecessary (and should not be here).  More importantly, the default mark color should not be "black".  Please revert this file.